python/cudnn/engines/matmul_cutile_engine.py

```
@@ -0,0 +1,221 @@
+            RuntimeError: If GPU or driver doesn't meet requirements
+            NotImplementedError: If graph contains unsupported operations
+        """
+        # Check GPU compute capability (need SM100+ for Blackwell)
+        err, device_id = cudart.cudaGetDevice()
+        err, props = cudart.cudaGetDeviceProperties(device_id)
+        cc_int = props.major * 10 + props.minor
+        if cc_int < 100:
+            raise RuntimeError(f"MatmulCuTileEngine requires Blackwell GPU (SM100+), " f"got SM{cc_int}")
+
+        # Check driver version (need r580+)
+        err, driver_version = cudart.cudaDriverGetVersion()
```

Ruff (0.15.20)

[warning] Unpacked variable `err` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)